Presentation.Web/Controllers/API/V2/External/ItSystemUsages/Mapping/ItSystemUsageResponseMapper.cs

```diff
@@ -175,7 +175,7 @@ private static GeneralDataResponseDTO MapGeneral(ItSystemUsage systemUsage)
                 Validity = new ValidityResponseDTO
                 {
                     EnforcedValid = systemUsage.Active,
-                    Valid = systemUsage.IsActive,
+                    Valid = systemUsage.IsActiveAccordingToDateFields,
```

Comment:
like we talked about yesterday, this should contain the "full truth", so create a new function on itsystemusage to provide the "Valid" flag and create a new DTO property `ValidAccordingToValidityPeriod`

Reply:
the reason for this is to get the combined validity for the api users as well as the details